packages/property-grid/package.json

```diff
@@ -1,33 +1,41 @@
 {
-  "name": "@bentley/property-grid-react",
-  "version": "2.2.4",
-  "description": "This package offers a React widget which provides access to the property grid within an iModel. Designed for use in an iModeljs 9-zone application",
+  "name": "@itwin/property-grid-react",
+  "version": "0.1.0",
```

Comment:
should this be a dev version?

Reply:
i think for this pkg an tree-widget, instead of doing pre-releases and its 0.x, just do minor bumps. need to talk this over with caleb before i publish.